components/lsp-form/helpers.ts

```diff
@@ -0,0 +1,13 @@
+import { DateTime, Duration } from "luxon";
+
+// Stub function to show effect.
+export const calculateTimeRemaining = () => {
```

Comment:
not sure if you want to flesh this out now or later, but we should instantiate with an end date